### PR TITLE
Corporate Liaison Playtime Ranks (Approved by Tivi)

### PR DESCRIPTION
**Pull request by Loyavas**
## About The Pull Request

Adds 5 different pay grades, with each one of them controlling the Corporate Liaison's "title"/rank, based on the playtime hours of each person's time as a Corporate Liaison, with: Starting - Corporate Intern - "Int", 10 hours - Corporate Associate - "Assoc", 25 hours - Corporate Partner - "Ptnr", 50 hours - Corporate Analyst - "Anlst", 100 hours - Corporate Supervisor - "Spvr"

## Why It's Good For The Game

While such an addition may seem pointless at first, in actuality, it rewards the very idea of playing the role of a Corporate Liaison, whereas it traditionally exists primarily to be shit on by marines. The whole idea is to implement a bloated corporate structure with arbitrary ranks that technically don't change permissions, however in actuality it is a good addition in order for overall encouragement of both interaction and "gimmick" roleplay, while also allowing individuals NOT associated or familiar with the idea of roleplaying liaisons, to be able to staunchly identify how much experience one shalt have gained over the time, and thus being an indicator of the respectability of a liaison, which in turn allows for a unique response to any gimmicks/roleplay offered or interjected, rather than the generic response of a majority of the playerbase seeing them all as useless and of the same generic quality.
The amount of hours is specified due to the general low amounts of playtime that are cranked out by the general playerbase, while still maintaining slightly mediocre sized sums to show the dedication of each Liaison, however they can be adjusted according to its needs and the maintainers' opinions

Also they look cool.

## Changelog
:cl:
add: Added 5 paygrades for Liaison
/:cl:

### CODE REVIEW

**Comment by MiniMeatwad** [comment]
Don't forget that CL stands for Clown.

**Comment by OpsCoretango** [comment]
okay, what advantages(everybody loves donuts and hotdogs) of ranks?

**Comment by Loyavas** [comment]
Accidentally added another commit to this, fixed and removed, pretend it never happened

**Comment by Derrick2Electricboogaloo** [comment]
> Accidentally added another commit to this, fixed and removed, pretend it never happened

I simply can not, day is now ruined.

**Review by TiviPlus — APPROVED** [review]
tldr

**Comment by AnCopper** [comment]
> Don't forget that CL stands for Clown.

It stands for crap licker.

**Comment by Licherr** [comment]
time to grind
